VultisigApp/VultisigApp/View Models/SwapCryptoViewModel.swift

```diff
@@ -191,7 +191,8 @@ class SwapCryptoViewModel: ObservableObject, TransferViewModel {
         guard tx.isApproveRequired, let spender = tx.router else {
             return nil
         }
-        let amount = tx.amountInCoinDecimal
+        // Add 10% buffer to prevent approval issues
+        let amount = tx.amountInCoinDecimal * BigInt(115) / BigInt(100)
```

Comment:
What kind of approval issues are you referring to?  can you please revert it?  
Like if I would like to swap 100USDT , then I should just approve 100USDT , not 115

Reply:
Let's investigate it more , we can't approve more than it need